src/Form/RadioGroup.js

```diff
@@ -87,7 +90,9 @@ class RadioGroup extends Component {
                   valueFalse={value}
                   iconOn="radiobox-marked"
```

Comment:
Can we add `iconOn` and `iconOff` as props while we're here and set defaultProps to these values?

Reply:
Sounds good, will update.

Reply:
Thanks, updated.